serving/src/main/java/feast/serving/service/CachedSpecStorage.java

```diff
@@ -177,4 +209,16 @@ public StorageSpec load(String key) throws Exception {
   public boolean isConnected() {
     return coreService.isConnected();
   }
+
+  /** Preload all spec into cache. */
+  public void populateCache() {
```

Comment:
With a MAX_SPEC_COUNT of 1000, once you have more than that, you're going to be evicting things that might actually be commonly used every minute. Is that a problem?

Reply:
Yeah, that's probably introduce problem. Do you think it's better to not limit the cache size based on the number of entry?